cdap-explore/src/test/java/co/cask/cdap/explore/service/HiveExploreServiceStreamTest.java

```diff
@@ -314,4 +359,26 @@ private static String getTableName(String streamName) {
     out.close();
     return out.toByteArray();
   }
+
+  private static void grantAndAssertSuccess(EntityId entityId, Principal principal, Set<Action> actions)
+    throws Exception {
+    Set<Privilege> existingPrivileges = authorizer.listPrivileges(principal);
+    authorizer.grant(entityId, principal, actions);
+    ImmutableSet.Builder<Privilege> expectedPrivilegesAfterGrant = ImmutableSet.builder();
+    for (Action action : actions) {
+      expectedPrivilegesAfterGrant.add(new Privilege(entityId, action));
+    }
+    Assert.assertEquals(Sets.union(existingPrivileges, expectedPrivilegesAfterGrant.build()),
+                        authorizer.listPrivileges(principal));
+  }
+
+  private static void revokeAndAssertSuccess(EntityId entityId, Principal principal, Set<Action> actions)
+    throws Exception {
+    Set<Privilege> existingPrivileges = authorizer.listPrivileges(principal);
+    authorizer.revoke(entityId, principal, actions);
+    for (Action action : actions) {
+      existingPrivileges.remove(new Privilege(entityId, action));
```

Comment:
Not a good idea to allow updates to the result of list privileges. Might cause unintentional bad tests.